resources/conf/config.toml

```diff
@@ -12,6 +12,10 @@
     username = "admin"
     password = "$env{adapter_admin_pwd}"
 
+[[adapter.vhost]]
+  environment = "Production and Sandbox"
+  vhost = "localhost"
+
```

Comment:
router.systemHost - health, testkey hostname
adapter.vhostMapping - standalone mgw deployment

Reply:
Add comments to configs

Reply:
changed configs and added comments for the configs with https://github.com/wso2/product-microgateway/pull/1826/commits/3aa9b7de891d72fbd25fdfb61f7dac23d0f11d99